cmd/kubeadm/app/phases/apiconfig/setupmaster.go

```diff
@@ -49,6 +50,8 @@ func findMyself(client *clientset.Clientset) (*v1.Node, error) {
 }
 
 func attemptToUpdateMasterRoleLabelsAndTaints(client *clientset.Clientset) error {
+	validate.WaitForNode(client)
```

Comment:
Actually yes, I think node readiness check must be removed. BTW is `validate.Validate()` dead code currently?